server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalHistogram.java

```diff
@@ -37,7 +37,7 @@
 /**
  * Implementation of {@link Histogram}.
  */
-public final class InternalHistogram extends InternalMultiBucketAggregation<InternalHistogram, InternalHistogram.Bucket>
+public class InternalHistogram extends InternalMultiBucketAggregation<InternalHistogram, InternalHistogram.Bucket>
```

Comment:
I think this is OK as none of our internal aggregations is final and removing it allows mocking the class.